pages/api/button.md

```diff
@@ -47,18 +47,18 @@ This property accepts the following keys:
 | <span class="prop-name">flatPrimary</span> | Styles applied to the root element for backwards compatibility with legacy variant naming.
 | <span class="prop-name">flatSecondary</span> | Styles applied to the root element for backwards compatibility with legacy variant naming.
 | <span class="prop-name">outlined</span> | Styles applied to the root element if `variant="outlined"`.
-| <span class="prop-name">contained</span> | Styles applied to the root element if `variant="[contained \| fab]"`.
-| <span class="prop-name">containedPrimary</span> | Styles applied to the root element if `variant="[contained \| fab]"` and `color="primary"`.
```

Comment:
Looks like we lost a closing bracket?

Reply:
thanks!

Reply:
https://spec.commonmark.org/0.26/#backslash-escapes

Reply:
I'm working on this if you don't mind 👼